code/modules/jobs/job_types/centcom.dm

```diff
@@ -0,0 +1,280 @@
+/datum/outfit/centcom/official //Generic centcom person. Whatever rank you want that is Lieutenant or lower.
+	name = "(CO-1)CentCom Official"
+
+	uniform = /obj/item/clothing/under/rank/centcom_officer
+	suit = null
+	shoes = /obj/item/clothing/shoes/sneakers/black
+	gloves = /obj/item/clothing/gloves/color/black
+	ears = /obj/item/radio/headset/headset_cent
+	glasses = /obj/item/clothing/glasses/sunglasses
+	head = /obj/item/clothing/head/beret/sec/centcom
+	belt = /obj/item/gun/energy/e_gun
+	l_pocket = /obj/item/pen
+	back = /obj/item/storage/backpack/satchel
+	r_pocket = /obj/item/pda/heads
+	l_hand = /obj/item/clipboard
+	id = /obj/item/card/id/centcom
+	implants = list(/obj/item/implant/mindshield)
+	backpack_contents = list(/obj/item/storage/box/hug/survival=1, /obj/item/restraints/handcuffs/cable/zipties=1)
+
+/datum/outfit/centcom/official/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	if(visualsOnly)
+		return
+
+	var/obj/item/pda/heads/pda = H.r_store
+	pda.owner = H.real_name
+	pda.ownjob = "CentCom Official"
+	pda.update_label()
+
+	var/obj/item/card/id/W = H.wear_id
+	W.icon = 'icons/obj/card.dmi' //Bypasses modularisation.
+	W.icon_state = "centcom"
+	W.access = get_centcom_access("CentCom Official")
+	W.access += ACCESS_WEAPONS
+	W.assignment = "CentCom Official"
+	W.registered_name = H.real_name
+	W.update_label()
+
+	H.ignores_capitalism = TRUE // Yogs -- Lets Centcom guys buy a damned smoke for christ's sake
```

Comment:
All of these are under  /datum/outfit/centcom  yet /datum/outfit/centcom itself is not used for anything

Make /datum/outfit/centcom have W.registered_name = H.real_name and possibly H.ignores_capitalism = TRUE since you use them on all of these

If you want an example you can see this you can find it in the new amber task force

Reply:
i didnt that originally but it ignores_capitalism didn't seem to work. ill check now if it actually works at all then submit the change